spark/src/main/scala/org/apache/spark/sql/comet/CometMetricNode.scala

```diff
@@ -79,6 +79,15 @@ object CometMetricNode {
         "total time (in ms) spent in this operator"))
   }
 
+  /**
+   * SQL Metrics for Comet native ScanExec
+   */
+  def scanMetrics(sc: SparkContext): Map[String, SQLMetric] = {
+    Map(
+      "cast_time" ->
+        SQLMetrics.createNanoTimingMetric(sc, "Total time for casting arrays"))
```

Comment:
"arrays" in "casting arrays" may be confused with Spark array type and considered as time spent on casting (Spark) "arrays".

Maybe "casting columns"?